pkg/fdbadminclient/mock/admin_client_mock.go

```diff
@@ -274,34 +274,34 @@ func (client *AdminClient) GetStatus() (*fdbv1beta2.FoundationDBStatus, error) {
 				Roles:         fdbRoles,
 			}
 		}
+	}
 
-		for _, processGroup := range client.additionalProcesses {
-			locality := map[string]string{
-				fdbv1beta2.FDBLocalityInstanceIDKey: string(processGroup.ProcessGroupID),
-				fdbv1beta2.FDBLocalityZoneIDKey:     string(processGroup.ProcessGroupID),
-			}
+	for _, processGroup := range client.additionalProcesses {
```

Comment:
The diff seems a little bit off, I basically moved the for loop `for _, processGroup := range client.additionalProcesses` out of the for loop over the `Pods`.